tempColorTokenBuild.ts

```diff
@@ -0,0 +1,166 @@
+  ],
+  [
+    'light-colorblind',
+    [...functionalColorFiles.light, `tokens/functional/color/exceptions-light-colorblind.json`],
+    [`tokens/base/color/light.json`, `tokens/base/color/light-colorblind.json`]
+  ],
+  [
+    'light-high-contrast',
+    [...functionalColorFiles.light, `tokens/functional/color/exceptions-light-high-contrast.json`],
+    [`tokens/base/color/light.json`, `tokens/base/color/light-high-contrast.json`]
+  ],
+  ['dark', [...functionalColorFiles.dark], [`tokens/base/color/dark.json`]],
+  [
+    'dark-dimmed',
+    [...functionalColorFiles.dark, `tokens/functional/color/exceptions-dark-dimmed.json`],
+    [`tokens/base/color/dark.json`, `tokens/base/color/dark-dimmed.json`]
+  ],
+  [
+    'dark-tritanopia',
+    [...functionalColorFiles.dark, `tokens/functional/color/exceptions-dark-tritanopia.json`],
+    [`tokens/base/color/dark.json`, `tokens/base/color/dark-tritanopia.json`]
+  ],
+  [
+    'dark-colorblind',
+    [...functionalColorFiles.dark, `tokens/functional/color/exceptions-dark-colorblind.json`],
+    [`tokens/base/color/dark.json`, `tokens/base/color/dark-colorblind.json`]
+  ],
+  [
+    'dark-high-contrast',
+    [...functionalColorFiles.dark, `tokens/functional/color/exceptions-dark-high-contrast.json`],
+    [`tokens/base/color/dark.json`, `tokens/base/color/dark-high-contrast.json`]
+  ]
+]
+
+const getStyleDictionaryConfig: StyleDictionaryConfigGenerator = (
+  outputName,
+  source,
+  include,
+  options = {}
+): StyleDictionary.Config => ({
+  source, // build the special formats
+  include,
+  parsers: [w3cJsonParser],
+  // formats need to be defined here but are used in platforms
+  format: {
+    'scss/mixin-css-variables': scssMixinCssVariables,
+    'javascript/esm': javascriptEsm,
+    'javascript/commonJs': javascriptCommonJs,
+    'typescript/export-definition': typescriptExportDefinition
+  },
+  // transforms need to be defined here but are used in platforms
+  transform: {
+    'color/rgbAlpha': colorToRgbAlpha,
+    'color/hexAlpha': colorToHexAlpha,
+    'color/hex6': colorToHex6
+  },
+  platforms: {
+    css: platformCss(`${outputName}.css`, undefined, BUILD_PATH, options),
+    scss: platformScss(`${outputName}.scss`, undefined, BUILD_PATH),
+    js: platformJs(`${outputName}.js`, undefined, BUILD_PATH),
+    ts: platformTs(`${outputName}.ts`, undefined, BUILD_PATH),
+    json: platformJson(`${outputName}.json`, undefined, BUILD_PATH),
+    docJson: platformDocJson(`${outputName}.json`, PREFIX, BUILD_PATH)
+  }
+})
+
+/**
+ * Copy `removed` files
+ */
+copyFilesAndFolders([`removed`], `tokens`, BUILD_PATH)
+
+/**
+ * convert colors
+ */
+for (const [theme, source, include] of themes) {
+  StyleDictionary.extend(getStyleDictionaryConfig(`color/${theme}`, source, include)).buildAllPlatforms()
+}
+// TODO: Remove once shadows that used to be in colors are implemented
+// eslint-disable-next-line no-console
+console.log('⚠️ Shadows are not implemented in tokens correctly')
```

Comment:
Nice 👍

Reply:
> This is looking really great. I'm relying on @rezrah's technical expertise for final review 🙌
> 
> Have we thought about a way we might be able to do a test/compare the values of the old color token set vs. this new one? Either compare the values, or visually render and compare (or maybe both).

Hey, this is complicated without putting to much work into it.

What I have done is a simple `diff` of the old and new output files (`sass`, `css`, `js`).
However it requires some changes in the old code as the output there is a bit mixed.

I would suggest we do this in a separate PR that we never merge but just use to surface those diffs.
Also keep in mind that this only works until we start changing the names.

@rezrah do you have a better idea?